scout/run.py

```diff
@@ -3446,6 +3600,32 @@ def compete_adj(
                     # on the basis of the factors calculated above
                     adj_t_e[var] = adj_t_b[var] * b_e_ratio * rp_adj[var]
 
+                    # In the special case of efficient energy use where
+                    # captured efficient energy is reported, develop a factor
+                    # that adjusts efficient-captured energy values to account
+                    # for missed competed stock in years before the measure was
+                    # on the market
+                    if var == "energy" and eff_capt:
+                        try:
+                            # Ratio to adjust efficient-captured to efficient
+                            # energy total
+                            e_ec_ratio = adj[var]["total"]["efficient"][yr] / \
+                                adj[var]["total"]["efficient-captured"][yr]
+                            # Ratio to determine what portion of efficient
+                            # energy was captured by measure since market entry
+                            # based on stock totals
+                            stk_capt_since_entry_ratio = \
+                                stk_capt_since_entry / stk_cmp_since_entry
+                        except (ZeroDivisionError, FloatingPointError):
+                            e_ec_ratio, stk_capt_since_entry_ratio = (
+                                1 for n in range(2))
```

Comment:
I know this is the typical notation used throughout, but can be simplified to
```suggestion
                            e_ec_ratio, stk_capt_since_entry_ratio = 1, 1
```

or `e_ec_ratio = stk_capt_since_entry_ratio = 1`